assets/js/linear_algebra/dot_product_collide2d.js

```diff
@@ -293,48 +302,42 @@ function compute(u, v){
   // We make a copy of uT and v table
   // Then we plot them at the same place as the original text
   // with opacity 1.0
-  let animation_begin = [
-      [
-          {text: ''}, {text: ''}, {text: ''}, {text: ''}, 
-          {text: 'v', text_opacity: 0.7, key: 'vv'},
-      ], [
-          {text: ''}, {text: ''}, {text: ''}, {text: ''},
-          {text: v.coord.x.toFixed(2), text_opacity: 0.7, key: 'xvv'},
-      ], [
-          {text: ''}, {text: ''}, {text: ''}, {text: ''},
-          {text: v.coord.y.toFixed(2), text_opacity: 0.7, key: 'yvv'},
-      ], [
-          {text: ''}, {text: ''}, {text: ''}, {text: ''}, 
-          {text: '', key: 'zvv'},
-      ], [
-          {text: 'u\u1d40', text_opacity: 0.7, key: 'uu'},
-          {text: u.coord.x.toFixed(2), text_opacity: 0.7, key: 'xuu'},
-          {text: u.coord.y.toFixed(2), text_opacity: 0.7, key: 'yuu'},
-          {text: '', key: 'zuu'},
-          {text: ''}
-      ]
-  ];
-
-  lib.plot_texts(lib.text_table_to_list(
-      animation_begin,
-      start_coord_x=start_coord_x, 
-      start_coord_y=start_coord_y,
-      w_unit=w_unit, h_unit=h_unit,
-      dws_array=dws_array,
-      dhs_array=dhs_array), tt=0, name='animation'
-  );
+  
+  let [lines_v, texts_v] = lib.text_matrix_to_list(
+          [[{text: v.coord.x.toFixed(2), key: 'xvv'}],
+           [{text: v.coord.y.toFixed(2), key: 'yvv'}]],
+          [last_col_coord, start_coord_y], 14
+          ),
+      [lines_uT, texts_uT] = lib.text_matrix_to_list(
+          [[{text: u.coord.x.toFixed(2), key: 'xuu'},
+            {text: u.coord.y.toFixed(2), key: 'yuu'}]],
+          [start_coord_x, last_row_coord], 14
+          );
+
+  let animation_begin = [],
+      lines_to_show = [];
+
+  animation_begin.push(...texts_uT);
+  animation_begin.push(...texts_v);
+  animation_begin.push(uT_cell, v_cell);
+
+  lines_to_show.push(...lines_uT);
+  lines_to_show.push(...lines_v);
+
+  lib.plot_texts(animation_begin, 0, 'animation');
+  lib.plot_lines(lines_to_show, 0, 'lines_to_show')
 
   // Immediately after that, plot them at the bottom right corner
   // with the same name & key, now with opacity 0.0
   let animation_end = [
-          {text: v.coord.x.toFixed(2), text_opacity: 0,tt: 900, key: 'xvv'},
-          {text: v.coord.y.toFixed(2), text_opacity: 0,tt: 450, key: 'yvv'},
+          {text: v.coord.x.toFixed(2), text_opacity: 0, tt: 900, key: 'xvv'},
+          {text: v.coord.y.toFixed(2), text_opacity: 0, tt: 450, key: 'yvv'},
           {text: u.coord.x.toFixed(2), text_opacity: 0, tt: 900, key: 'xuu'},
           {text: u.coord.y.toFixed(2), text_opacity: 0, tt: 450, key: 'yuu'},
   ];
   for (i = 0; i < animation_end.length; i++) {
-      animation_end[i].x = bot_right_x;
-      animation_end[i].y = bot_right_y;
+      animation_end[i].x = last_col_coord;
+      animation_end[i].y = last_row_coord + 0.09 * h_unit;
   };
   
   lib.plot_texts(animation_end, tt=0, name='animation');
```

Comment:
fix this argument name, no argument name passing in js

Reply:
done